tools/image-builder/build_bond.zsh

```diff
@@ -0,0 +1,82 @@
+#!/bin/zsh
+
+set -eux
+setopt shwordsplit
+
+HOME=$1
+FLAVOR=$2
+BOOST=${3:-}
```

Comment:
Any thoughts on setting an explicit default here? i.e., `BOOST=${3:-1.63.0}`.

Reply:
Talked offline. Will not add default and instead will have explicit check for cpp.